src/main/resources/cql-queries/downsample_insert_query.cql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO %s
```

Comment:
What is the idea behind storing the queries in files? Changing the query shouldn't be something we do unless we want to deploy an new release so it's just one extra step that will take away from performance.

Reply:
actually it was @harageth 's idea to store cql queries into template files as writing them into code doesn't look good and managing them could be messy.